views/index.html

```diff
@@ -1,8 +1,22 @@
 {{define "head"}}
+    <head>
+        {{ if .cssFiles}}
+            {{range .cssFiles}}
+                <link href={{ . }} rel="stylesheet">
+            {{end}}
+        {{end}}
+    </head>
```

Comment:
better to put it in master.html because we need apply CSS from baseapp to SSR UI components